app/models/petition_creator.rb

```diff
@@ -139,7 +139,12 @@ def save
 
       unless rate_limit.exceeded?(@petition.creator)
         @petition.save!
-        send_email_to_gather_sponsors(@petition)
+
+        if Site.minimum_number_of_sponsors == 0
```

Comment:
Wonder whether add a method to `Site` called `collecting_sponsors?` makes this more readable to someone in the future?, e.g.

``` ruby
class Site < ActiveRecord::Base
  class << self
    def collecting_sponsors?
      minimum_number_of_sponsors > 0
    end
  end
end
```